test/testAnimation.cpp

```diff
@@ -60,10 +60,32 @@ TEST_CASE("Animation Lottie", "[tvgAnimation]")
     REQUIRE(picture->load(TEST_DIR"/invalid.json") == Result::InvalidArguments);
     REQUIRE(picture->load(TEST_DIR"/test.json") == Result::Success);
 
+    REQUIRE(animation->totalFrame() == Approx(120).margin(004004));
```

Comment:
004004 ? we have this value in few places in this test

Reply:
Just one sample point. Its not the main point in this PR. Please let us not get side tracked.